locale/fi/radiobutton.json

```diff
@@ -0,0 +1,69 @@
+{
+  "title": "Valintanappi (Radio Button)",
+  "intro": "Valintanappi on lomakekomponentti, jolla voi valita yhden vaihtoehdon useammasta valintanapista koostuvasta valintanappiryhmästä. Käyttäjä voi valita kerrallaan vain yhden vaihtoehdon. Kun käyttäjä tekee uuden valinnan, aiemmin tehty valinta poistuu ja korvautuu uudella valinnalla.",
+  "note.title": "Saavutettavuus ja käytettävyys",
+  "note.items": [
+    "Pyri kuvaaviin, lyhyisiin ja ytimekkäisiin otsake-teksteihin (label).",
+    "Sijoita saman aihealueen valintanapit ryhmiin asemoimalla ne päällekkäin.",
+    "Käytä valintanappia, kun käyttäjän halutaan valitsevan yksi annetuista vaihtoehdoista.",
+    "Älä käytä valintanappia tilanteissa, joissa käyttäjän on määrä valita useita vaihtoehtoja. Käytä tällöin valintaruutua (checkbox).",
+    "Älä käytä valintanappia tilan muutokseen. Käytä tähän tarkoitukseen valintakytkintä (toggle).",
+    "Käytä ohjetekstiä tilanteissa, joissa käyttäjälle on tarpeen antaa lisäohjeistusta valinnan suhteen. Yksinkertaisissa tapauksissa ohjetekstiä ei tarvita.",
+    "Valittuun valintanappiin voi liittyä lisävalintoja, kuten esimerkiksi tekstikenttiä tai valintakytkimiä.",
+    "Lisävalinnat ovat piilotettuina oletustilassa ja ne näytetään vasta, kun valintanappi valitaan.",
+    "Lisävalinnat asetellaan suoraan valitun valintanapin alle sisennettynä."
+  ],
+  "sections": [
+    {
+      "title": "Mistä komponentti koostuu?",
```

Comment:
The question mark doesn't belong here I think.

Reply:
I think it can be used, but have to check how this is done elsewhere to keep things uniform. https://www.kielikello.fi/-/otsikot-ojennukseen

Reply:
Looks like it's not used elsewhere, so will remove.